lib/content-services/src/lib/dialogs/folder.dialog.html

```diff
@@ -1,4 +1,4 @@
-<h2 mat-dialog-title>
+<h2 mat-dialog-title id="title">
```

Comment:
I would use a class instead of ID as it can be reusable later on

Reply:
using IDs to style components is an antipattern, please use class names

Reply:
replaced id with class.